### DISCUSSION TITLE
httpx shouldn't output color codes to output file

### DISCUSSION THREAD

**Discussion by meme-lord** [projectdiscovery/httpx#543] [General] [answered]
### httpx version: 1.2.0
httpx sends the color codes to the output file (specified by command argument) and I think it would make more sense if it didn't.
If someone wanted the color codes in the output file they could just do:
`echo 'google.com' | httpx -title >  out.txt`

I understand that one could use -nc to get no colour but I think what I am suggesting is a better default.

### Current Behavior:
( -title is just being used as an example, this happens with other options also)
`echo 'google.com' | httpx -o out.txt -title`
out.txt contains:
`https://google.com [ESC[36m301 MovedESC[0m]`

### Expected Behavior:
`echo 'google.com' | httpx -o out.txt -title`
out.txt contains:
`https://google.com [301 Moved]`

**Comment by ehsandeep** [comment, marked as answer]
@meme-lord adding `~/.config/httpx/config.yaml` should do the trick to default any of your flags settings.

```
# disable color in output
no-color: true
```
